themes/sd/css/base.css

```diff
@@ -144,11 +144,10 @@ footer .container > div:last-child {
  * Top Navigation
  */
 .navbar {
-    background: url('/img/main_bkgrd_home.png') no-repeat;
-    background-size: cover;
+    background: #333;
 }
 .navbar-brand {
-    background: transparent url('/svg/Screwdriver_Logo_White.svg') no-repeat;
```

Comment:
Should we delete this file? It doesn't look like we're using Screwdriver_Logo_White anywhere if this is gone.